Google.Cloud.AspNetCore/Google.Cloud.AspNetCore.Firestore.DistributedCache/FirestoreCacheExtensions.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) 2019 Google LLC.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+using System;
+using Microsoft.Extensions.Caching.Distributed;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace Google.Cloud.AspNetCore.Firestore.DistributedCache
+{
+    public static class FirestoreCacheExtensions
+    {
+        /// <summary>
+        /// Add a distributed cache that stores cache entries in Firestore.
+        /// </summary>
+        /// <param name="projectId">Your Google Cloud Project Id.
+        /// If null, pulls your project id from the current application
+        /// default credentials.
+        /// </param>
+        public static IServiceCollection AddFirestoreDistributedCache(

```

Comment:
I expect you'll get warnings (or at least, I usually do) if you document some parameters but not others - so `services` needs documentation too.
There may be an exception for the target of an extension method...

Reply:
Done.